components/clm/src/biogeochem/CNAllocationMod.F90

```diff
@@ -2963,6 +2964,8 @@ subroutine CNAllocation3_PlantCNPAlloc (bounds            , &
          npool_to_grainn_storage      => nitrogenflux_vars%npool_to_grainn_storage_patch     , & ! Output: [real(r8) (:)   ]  allocation to grain N storage (gN/m2/s)
          retransn_to_npool            => nitrogenflux_vars%retransn_to_npool_patch           , & ! Output: [real(r8) (:)   ]  deployment of retranslocated N (gN/m2/s)
          sminn_to_npool               => nitrogenflux_vars%sminn_to_npool_patch              , & ! Output: [real(r8) (:)   ]  deployment of soil mineral N uptake (gN/m2/s)
+         nfix_to_plantn               => nitrogenflux_vars%nfix_to_plantn_patch              , &
+         biochem_pmin_to_plant        => phosphorusflux_vars%biochem_pmin_to_plant_patch     , &
```

Comment:
It might be helpful for code readability to add comments to these two lines (in analogy to the lines surrounding them).